xbmc/cores/VideoPlayer/AudioSinkAE.cpp

```diff
@@ -356,7 +356,10 @@ CAEStreamInfo::DataType CAudioSinkAE::GetPassthroughStreamType(AVCodecID codecId
       break;
 
     case AV_CODEC_ID_DTS:
-      format.m_streamInfo.m_type = CAEStreamInfo::STREAM_TYPE_DTSHD;
+      if (profile == FF_PROFILE_DTS_HD_HRA)
+        format.m_streamInfo.m_type = CAEStreamInfo::STREAM_TYPE_DTSHD;
+      else
```

Comment:
is there any way this could not be `FF_PROFILE_DTS_HD_MA `? ie. should we be explicitly checking for that here?

Reply:
Nope. In fact we have two possibilities we can detect: those that need 192 / 8 and those that work with 192 / 2.

As FFMPEG in our version can only distinguish between HRA and MA - we use everything that we don't understand in the else clause.

Or in other words: if it's exactly HRA, we can use DTSHD, else sink must support DTSHD_MA